app/models/ability.rb

```diff
@@ -57,6 +57,14 @@ def initialize(user)
       end
     end
 
+    can :manage_governing_apo, ActiveFedora::Base do |dor_item, new_apo_id|
+      next false unless dor_item.can_manage_item?(user.roles(new_apo_id))
```

Comment:
no longer needed, because the whole thing was compacted into a single `&&` conditional once the `if dor_item.admin_policy_object ...` part got turned into `can?(:manage_item, dor_item)`